src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,88 @@
 import React from 'react';
+import classNames from 'classnames';
 import './TodoList.scss';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
+export const TodoList = ({ text,
```

Comment:
```suggestion
export const TodoList = ({ 
  text,
```
move all props to the new lines